example-context/example-service/example-event/example-event-processor/src/main/java/uk/gov/justice/services/example/cakeshop/event/processor/CakeMadeEventProcessor.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.services.example.cakeshop.event.processor;
+
+import static uk.gov.justice.services.core.annotation.Component.EVENT_PROCESSOR;
+
+import uk.gov.justice.services.core.annotation.Handles;
+import uk.gov.justice.services.core.annotation.ServiceComponent;
+import uk.gov.justice.services.core.sender.Sender;
+import uk.gov.justice.services.messaging.JsonEnvelope;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@ServiceComponent(EVENT_PROCESSOR)
+public class CakeMadeEventProcessor {
+
+    Logger logger = LoggerFactory.getLogger(CakeMadeEventProcessor.class);
+
+    @Inject
+    Sender sender;
+
+
+
+    @Handles("cakeshop.cake-made")
+    public void handle(final JsonEnvelope envelope) {
```

Comment:
I 'think' we were calling the name of the envelope variable according to what it actually is, so in this case 'event', although that might not have been updated everywhere.